jni/src/main/java/com/horizen/schnorrnative/SchnorrSignaturesList.java

```diff
@@ -0,0 +1,80 @@
+package com.horizen.schnorrnative;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SchnorrSignaturesList implements AutoCloseable {
+    private SchnorrSignature[] updatedSigningKeysSkSignatures;
+    private SchnorrSignature[] updatedSigningKeysMkSignatures;
+    private SchnorrSignature[] updatedMasterKeysSkSignatures;
+    private SchnorrSignature[] updatedMasterKeysMkSignatures;
```

Comment:
So here: if there was any signature update for a key at index i, then that position in these vectors must be set. Otherwise, it must be set to none (new SchnorrSignature()), reflecting the same KEYS order (if we unify the two classes this will come easier)